pkg/providers/ensure_updated.go

```diff
@@ -18,6 +18,10 @@ import (
 func EnsureUpdated(ctx context.Context, cc *grpc.ClientConn, verbose bool) error {
 	client := teamv1alpha1.NewTeamServiceClient(cc)
 
+	// client.ListMembers()
+	// @TODO: make me work!
+	// client.ListRoles()
```

Comment:
nitpick we can remove these
```suggestion
	// client.ListMembers()
	// @TODO: make me work!
	// client.ListRoles()
```